package.json

```diff
@@ -49,7 +49,8 @@
     "ts-node": "10.9.1",
     "tslint": "5.5.0",
     "typescript": "^4.7.4",
-    "vsce": "2.15.0"
+    "vsce": "2.15.0",
+    "ovsx": "latest"
```

Comment:
Not sure if this is required to add to this file. ovsx is used in the GHA where we publish extensions in oepnvsx: "npx ovsx publish". npx will automatically get the latest version of ovsx for each execution.